### PR TITLE
Review one of your past projects

### PR DESCRIPTION
**Pull request by SlowKingV**
*Thanks to @AmaliaGM for lending me her code for this project!!*

## Project requirements
- Create a new empty repo.
- Copy your code from the project that you completed during the Admissions Trial: - If you do not have the code from the Admissions Trial ask someone from your Standup Team to share their code with you. In that case, remember to add an attribution note and acknowledgment in your README file.
- Create a new branch for your copied code.
- Open a pull request.
- Play the role of the Code Reviewer:
  - Follow the list from the article ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).
  - Check if you have used [semantic HTML tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.).
  - Check if you applied [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!
- Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. (The only difference will be that the Code Reviewer won't check your code but the review you performed - isn't it a code review inception? ;))

### CODE REVIEW

**Review by SlowKingV — COMMENTED** [review]
- Kindly remember to use only two spaces for indentation in your `.html` and `.css` files
- I'd suggest making your code is correctly indented. Make sure the opening tag is at the same level as the closing tag and every child is one level (two spaces) more to the right
- Try to fix all linter errors you get

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="navbar">
+        <a href="#home" class="active">Home</a>
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+        <a href="javascript:void(0);" class="videos" onclick="myFunction()"></a>
+            <i class="showvideos"></i>
+    </div>
```
Comment (SlowKingV) [inline index.html:17]: I'd suggest using HTML5 semantic tags!
Instead of `<div>` try to use `<nav>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   <h1>Regression Analysis</h1>
+
+    <br>
+    <div style="text-align:start; margin-left: 10%;">
+    <iframe  width="727" height="409" src="https://www.youtube.com/embed/0m-rs2M7K-Y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </div>
+
+    <br>
+    <div class="videonavbar">
+        <a class="icon" href="views"><img src="https://cdn-icons.flaticon.com/png/128/3049/premium/3049489.png?token=exp=1652361818~hmac=6a0626f789c478dd89bd967469df035e" alt="Views" height="5%" width="5%"></a>
```
Comment (SlowKingV) [inline index.html:30]: I'd suggest reviewing the image as it is not showing up

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    </div>
+    <br>
+    <div class="sidenav">
+        <a href="#"><iframe width="75%" height="50%" src="https://www.youtube.com/embed/esskJJF8pCc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></a>
+        <br>
+        <a href="#"><iframe width="75%" height="50%" src="https://www.youtube.com/embed/Ollp2nSQCLY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></a>
+        <br>
+        <a href="#"><iframe width="75%" height="50%" src="https://www.youtube.com/embed/29rjWClT_3U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></a>
+        <br>
+      </div>
```
Comment (SlowKingV) [inline index.html:42]: I'd suggest using HTML5 semantic tags!
Instead of `<div>` try to use `<aside>`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+      <div class="main">
+         </div>
```
Comment (SlowKingV) [inline index.html:45]: Kindly avoid leaving empty elements if they don't have a design purpose

File: index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        
+        With a Regression analysis you can measure the influence of one variable or several variables on another variable, or you can predict a variable based on other variables.
+        </p>
+</body>
+<br>
+<footer>
+<div id="contact">
+    <h4> CONTACT: </h4>
+    <ul>
+      <img src="https://cdn-icons.flaticon.com/png/128/3955/premium/3955024.png?token=exp=1652369583~hmac=ba6bbe116341d4d665654fea4f1b3f70" alt="Instagram" width="5%" height="5%">
```
Comment (SlowKingV) [inline index.html:57]: I'd suggest reviewing the image as it is not showing up

File: index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      <img src="https://cdn-icons-png.flaticon.com/128/5968/5968764.png" alt="Facebook" width="5%" height=5%"> 
+     
+      <img src="https://cdn-icons-png.flaticon.com/128/733/733579.png" alt="Instagram" width="5%" height="5%">
+    </ul></div>
+<div>
+    <h4 id="about">
+        ABOUT:  Maria Amalia Gomez Moro
+    </h4>
+</div>    
+</footer>
```
Comment (SlowKingV) [inline index.html:68]: Make sure to not put any elements outside the page's `<body>`!

File: style.css [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+  }
+  #contact{
+    list-style-type: none; 
+    font-family: 'Courier New', Courier, monospace;
+    text-align:center;
+    margin: 10px
+    
+  
+    
+  }
```
Comment (SlowKingV) [inline style.css]: I'd suggest moving any `@media { ... }` rule-set to the end of the document to avoid undesired overwriting problems

**Review by algerina — COMMENTED** [review]
Hi @SlowKingV , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project where the linters are not passing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by SlowKingV** [comment]
@algerina  Sorry for the mention!
I am a little confused. Do I have to fix linters when doing the code review project? I thought you would only review my review (if that makes sense haha), especially since this isn't my code but the code I asked my coding partner to review!
This is the project I'm doing right now > https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md

**Comment by SlowKingV** [comment]
I took the liberty to fix the linter errors but my question on my [previous comment](https://github.com/SlowKingV/code-review/pull/1#issuecomment-1152783712) remains

**Comment by SlowKingV** [comment]
Sorry but this error is showing up on stylelint and I don't know what to do (I think it's a GitHub error because it is working just fine on my end) but all the linter errors have been fixed
Screenshot_8

File: index.html (hunk 1 above)
Comment (algerina) [reply index.html:17]: - [ ] (Optional) You did a great job, however kindly remember to always answer the "Why" are you requiring these changes.

**Review by algerina — APPROVED** [review]
Hi @SlowKingV , 
Good job so far! 

May I highlight the following:

- The repo looks professional. :heavy_check_mark: 
- You added valuable comments  :heavy_check_mark: 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**